Update version to v0.3.0 in examples (#453)
yorkie-team/yorkie-js-sdk: MAINTAINING.md

3. Create Pull Request and merge it into main.
4. Create [a new release](https://github.com/yorkie-team/yorkie-js-sdk/releases/new) by attaching the changelog by clicking `Generate release notes` button.
5. Then [GitHub action](https://github.com/yorkie-team/yorkie-js-sdk/blob/main/.github/workflows/npm-publish.yml) will deploy Yorkie JS SDK to [npm](https://www.npmjs.com/package/yorkie-js-sdk).

> 6. Update Yorkie version of example codes in `/examples`. Edit Yorkie version in `package.json` at each example project. Create Pull Request again and merge it into main.

Reviewer: 6. Update `yorkie-js-sdk` version in each project's `package.json` in [/examples](https://github.com/yorkie-team/yorkie-js-sdk/tree/main/examples) and submit a new Pull Request and merge it into the main branch.

> 6. Update Yorkie version of example codes in `/examples`. Edit Yorkie version in `package.json` at each example project. Create Pull Request again and merge it into main.

Reviewer: ```suggestion
6. Update `yorkie-js-sdk` version in each project's `package.json` in [/examples](https://github.com/yorkie-team/yorkie-js-sdk/tree/main/examples) and submit a new Pull Request and merge it into the main branch.
```
